omnibus/config/software/datadog-agent.rb

```diff
@@ -31,9 +31,9 @@
   env = with_embedded_path(env)
 
   # we assume the go deps are already installed before running omnibus
-  command "invoke agent.build --rebuild --use-embedded-libs --no-development", env: env
+  command "inv -e agent.build --rebuild --use-embedded-libs --no-development --embedded-path=#{install_dir}/embedded", env: env
```

Comment:
instead of relying on `pkg-config`, we inject the path to the embedded libs from here

Reply:
Thank you!